scripts/clang_tidy_wrapper.sh

```diff
@@ -8,10 +8,11 @@ file=$1
 shift  # Remove the file from $@
 
 file_relative_to_source=${file//$cmake_source_dir/}
-
+file_relative_to_source=${file_relative_to_source/..\/src/src}
```

Comment:
I'd guess there is some magic hidden with this double slash that I do not understand because bash.
I guess a simple comment would help here a lot.

Reply:
done